tests/tests_simple_settings.yml

```diff
@@ -16,14 +16,7 @@
     - name: apply the settings - call the role
       include_role:
         name: linux-system-roles.kernel_settings
-
-    # NOTE: This is only to include the vars from vars/main.yml - the
-    # when: false will skip all of the tasks in the role except the
-    # implicit tasks to load the vars and defaults
```

Comment:
I would prefer to keep a comment explaining why `public: true` is needed (what variable from the role do we need later).

Reply:
fixed in https://github.com/linux-system-roles/kernel_settings/pull/29/commits/4f14c744e9988aca05d72e4cf0c9ae3b7e29a459